Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -252,16 +296,13 @@ theorem mk₂_apply (f : M → Nₗ → Pₗ) {H1 H2 H3 H4} (m : M) (n : Nₗ) :
 
 variable {R}
 
-variable (f : M →ₛₗ[σ₁₃] N →ₛₗ[σ₂₃] P)
 
 variable (A Pₗ)
 variable [Module A Pₗ] [SMulCommClass R A Pₗ]
 
 /-- Composing a given linear map `M → N` with a linear map `N → P` as a linear map from
 `Nₗ →ₗ[R] Pₗ` to `M →ₗ[R] Pₗ`. -/
-def lcomp (f : M →ₗ[R] Nₗ) : (Nₗ →ₗ[R] Pₗ) →ₗ[A] M →ₗ[R] Pₗ :=
-  letI := SMulCommClass.symm
-  flip <| LinearMap.comp (flip id) f
+def lcomp (f : M →ₗ[R] Nₗ) : (Nₗ →ₗ[R] Pₗ) →ₗ[A] M →ₗ[R] Pₗ := lcompₛₗ _ _ _ f
```

Comment:
Hmm, `R` is not generalized to Semiring here. I have to do it again in #24015 ...

Reply:
Good catch